CHANGELOG.md

```diff
@@ -3,6 +3,51 @@
 All notable changes to this project will be documented in this file.
 See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.
 
+## 4.0.0-alpha.37
+
+_July 12, 2021_
+
+Big thanks to the 7 contributors who made this release possible. Here are some highlights ✨:
+
+- 🔎 Add the \"is empty\" and \"is not empty\" filter operators to date and number columns (#2274) @flaviendelangle
+- ⚡️ Only support @material-ui/core 4.12.0 or higher (#2281) @DanailH
+- 🐞 Fix a regression to not require @material-ui/x-license when using the DataGrid (#2295) @oliviertassinari
+- 👁️ Add `onViewportRowsChange` prop (#2038) @DanailH
+- 📃 Translate booleans when exporting rows to CSV (#2296) @m4theushw
+
```

Comment:
Does it make sense?

```suggestion

This is the last alpha release. We are moving to beta in the next release, next week.

```

Reply:
If we're allowed to make breaking changes during the beta phase, I'm OK with this announcement. #2312 still has work to be done.